---
title: Driver Projects for PRECISE SG100K
permalink: /collaborators-driver-projects-for-precise-sg100k/
variant: tiptap
description: ""
third_nav_title: Our Data Users
---
<h3><strong>Computation of Genome - Wide LD Scores and Matrices from the SG100K resource</strong></h3>
<table style="minWidth: 50px">
<colgroup>
<col>
<col>
</colgroup>
<tbody>
<tr>
<th rowspan="1" colspan="1">
<p>Team</p>
</th>
<th rowspan="1" colspan="1">
<p>Aims of Project</p>
</th>
</tr>
<tr>
<td rowspan="1" colspan="1">
<ul data-tight="true" class="tight">
<li>
<p><strong>Lead PI: Li Jingmei</strong>
<br>A*STAR Genome Institute of Singapore</p>
</li>
<li>
<p><strong>Co-Lead PI: Rajkumar s/o Dorajoo</strong>
<br>A*STAR Genome Institute of Singapore</p>
</li>
<li>
<p><strong>Co-Lead PI: Khor Chiea Chuen </strong>A*STAR Genome Institute
of Singapore</p>
</li>
</ul>
</td>
<td rowspan="1" colspan="1">
<p>Taking reference from similar work performed by the Pan-UK Biobank, the
team will compute in-sample dosage-based LD matrices and scores for each
of the three major ancestry groups in SG100K:</p>
<ul data-tight="true" class="tight">
<li>
<p>LD score regression analysis to estimate heritabilities</p>
</li>
<li>
<p>Fine-mapping analysis to identify causal variants of well-powered complex
traits</p>
</li>
</ul>
<p>The plan is to make the LD matrices available in Hail's BlockMatrix format
or similar. LD scores are also made available in LDSC-compatible flat files
(.l2.ldscore.gz and .M_5_50).</p>
</td>
</tr>
</tbody>
</table>
<p><strong><em>Institutions involved:</em></strong><em> A*STAR Genome Institute of Singapore</em>
</p>
<hr>
<h3><strong>Chronic Liver Disease is a Significant Risk Factor for Adverse Cardiometabolic Outcomes</strong></h3>
<table style="minWidth: 50px">
<colgroup>
<col>
<col>
</colgroup>
<tbody>
<tr>
<th rowspan="1" colspan="1">
<p>Team</p>
</th>
<th rowspan="1" colspan="1">
<p>Aims of Project</p>
</th>
</tr>
<tr>
<td rowspan="1" colspan="1">
<ul data-tight="true" class="tight">
<li>
<p><strong>Lead PI: Mark Chan</strong>
<br>National University Hospital
<br>Cardiology</p>
</li>
<li>
<p><strong>Co-Lead PI: Dr Nicholas Chew</strong>
<br>National University Hospital
<br>Cardiology</p>
</li>
</ul>
</td>
<td rowspan="1" colspan="1">
<ol data-tight="true" class="tight">
<li>
<p>Investigate associations between established non-invasive Chronic Liver
Disease (CLD) biomarkers and cardiometabolic outcomes.</p>
</li>
<li>
<p>Evaluate how these associations relate to major adverse cardiac events.</p>
</li>
<li>
<p>Examine whether these associations with CLD are independent from associated
metabolic disease.</p>
</li>
</ol>
</td>
</tr>
</tbody>
</table>
<p><strong><em>Institutions involved:</em></strong><em> National University Hospital</em>
</p>
<hr>
<h3><strong>Nonlinear Methods for Genomic Association Aanalysis of Eye Diseases</strong></h3>
<table style="minWidth: 50px">
<colgroup>
<col>
<col>
</colgroup>
<tbody>
<tr>
<th rowspan="1" colspan="1">
<p>Team</p>
</th>
<th rowspan="1" colspan="1">
<p>Aims of Project</p>
</th>
</tr>
<tr>
<td rowspan="1" colspan="1">
<ul data-tight="true" class="tight">
<li>
<p><strong>Lead PI: Liu Dianbo</strong>
<br>National University of Singapore
<br>Ophthamology</p>
</li>
<li>
<p><strong>Co-Lead PI: A/P Wee Hwee Lin</strong>
<br>National University of Singapore</p>
</li>
<li>
<p><strong>Co-Lead PI: Dr Nicolas Bertin</strong>
<br>A*STAR Genome Institute of Singapore</p>
</li>
</ul>
</td>
<td rowspan="1" colspan="1">
<ol data-tight="true" class="tight">
<li>
<p>Identify non-linear genetic associations contributing to \ the susceptibility
and manifestation of diverse eye diseases.</p>
</li>
<li>
<p>Explore epistatic interactions and allelic heterogeneity within the genomic
data to unravel the complex relationships between multiple genetic variants.</p>
</li>
<li>
<p>Investigate how non-linear responses to environmental variables contribute
to the phenotypic variation, with a focus on refining our understanding
of gene- environment interactions in the context of ocular health.</p>
</li>
<li>
<p>Investigate and interpret the biological relevance of non-linear genetic
associations. Aim to gain insights into the underlying mechanisms linking
identified genetic variants to specific eye diseases and contribute to
a more comprehensive understanding of the biology involved.</p>
</li>
<li>
<p>Evaluate the public health implications of the identified non-linear genetic
associations, considering their potential impact on disease prevention,
intervention, and personalised treatment strategies. Assess the translational
potential of the research findings to inform clinical practice, public
health policies, and contribute to advancements in precision medicine for
ocular health.</p>
</li>
</ol>
</td>
</tr>
</tbody>
</table>
<p><strong><em>Institutions involved:</em></strong><em> National University of Singapore</em>
</p>
<hr>
<h3><strong>Advancing the Understanding of Biological Mechanisms Influencing Chronic Inflammatory Skin Diseases</strong></h3>
<table style="minWidth: 50px">
<colgroup>
<col>
<col>
</colgroup>
<tbody>
<tr>
<th rowspan="1" colspan="1">
<p>Team</p>
</th>
<th rowspan="1" colspan="1">
<p>Aims of Project</p>
</th>
</tr>
<tr>
<td rowspan="1" colspan="1">
<ul data-tight="true" class="tight">
<li>
<p><strong>Lead PI: Yew Yik Weng</strong>
<br>National Skin Centre</p>
</li>
<li>
<p><strong>Co-Lead PI: Steven Thng Tien Guan</strong>
<br>National Skin Centre</p>
</li>
<li>
<p><strong>Co-Lead PI: Marie Loh</strong>
<br>Lee Kong Chian School of Medicine</p>
</li>
</ul>
</td>
<td rowspan="1" colspan="1">
<ol data-tight="true" class="tight">
<li>
<p>Identify host genetic factors associated with chronic inflammatory skin
diseases, specifically AD, psoriasis and chronic urticaria using genome
wide association and rare variant analyses among SG100K study participants,
taking advantage of whole genome sequence data and linkage to disease information
from national electronic health records (NEHR).</p>
</li>
<li>
<p>Examine the relationship between genetic variants and polygenic risk scores
(PRS) associated with skin phenotypes and real-world health data for skin
diseases (including diagnosis, onset, severity and treatment outcomes)
to identify genetic predictors of disease trajectories, complications and
co-morbidities and treatment outcomes using the TRUST dataset.</p>
</li>
</ol>
</td>
</tr>
</tbody>
</table>
<p><strong><em>Institutions involved:</em></strong><em> National skin Centre, Lee Kong Chian School of Medicine, A*STAR Skin Research Institute of Singapore</em>
</p>
<hr>
<h3><strong>Mood and Diet in Patients with Irritable Bowel Syndrome (IBS) in Singapore</strong></h3>
<table style="minWidth: 50px">
<colgroup>
<col>
<col>
</colgroup>
<tbody>
<tr>
<th rowspan="1" colspan="1">
<p>Team</p>
</th>
<th rowspan="1" colspan="1">
<p>Aims of Project</p>
</th>
</tr>
<tr>
<td rowspan="1" colspan="1">
<ul data-tight="true" class="tight">
<li>
<p><strong>Lead PI: Theresia Mina</strong>
<br>Lee Kong Chian School of Medicine</p>
</li>
<li>
<p><strong>Co-Lead PI: Jojn Chambers</strong>
<br>Lee Kong Chian School of Medicine</p>
</li>
<li>
<p><strong>Co-Lead PI: Sim Xueling</strong>
<br>National University of Singapore</p>
</li>
</ul>
</td>
<td rowspan="1" colspan="1">
<ol data-tight="true" class="tight">
<li>
<p>Evaluate the dietary pattern and characteristics of patients with IBS
in Singapore.</p>
</li>
<li>
<p>Identify patterns of mood disorders in patients with IBS in Singapore.</p>
</li>
<li>
<p>Identify genetic variants associated with IBS in the multi-ethnic Singaporean
Population. Explore and characterise common genetic polymorphisms IBS within
the diverse Singaporean population. This comprehensive genetic investigation
aims to unravel the unique genetic landscape of IBS, considering the multi-ethnic
composition of the population.</p>
</li>
<li>
<p>Investigate the effects of lifestyle factors on IBS Risk and progression.
Systematically examine the impact of lifestyle factors, including diet,
physical activity, sleep, stress, and mental health, on the risk and progression
of IBS. This multifaceted investigation seeks to discern the intricate
relationships between lifestyle choices and IBS, contributing valuable
insights for developing targeted interventions and improving patient outcomes.</p>
</li>
<li>
<p>Elucidate potential interactions between genetic and environmental influences
on IBS. Uncover and elucidate potential interactions between genetic factors
and environmental influences in the development and progression of IBS.
This integrated approach aims to provide a nuanced understanding of how
genetic predispositions and environmental exposures collaboratively contribute
to the manifestation of IBS, offering a foundation for personalised and
precision medicine strategies.</p>
</li>
</ol>
</td>
</tr>
</tbody>
</table>
<p><strong><em>Institutions involved:</em></strong><em> Tan Tock Seng Hospital, Lee Kong Chian School of Medicine</em>
</p>
<hr>
<h3><strong>The Contribution of Genetics to Dietary Habit and Its Relation to Adiposity and Cardiometabolic Diseases in Multiethnic Asian Population</strong></h3>
<table style="minWidth: 50px">
<colgroup>
<col>
<col>
</colgroup>
<tbody>
<tr>
<th rowspan="1" colspan="1">
<p>Team</p>
</th>
<th rowspan="1" colspan="1">
<p>Aims of Project</p>
</th>
</tr>
<tr>
<td rowspan="1" colspan="1">
<ul data-tight="true" class="tight">
<li>
<p><strong>Lead PI: Theresia Mina</strong>
<br>Lee Kong Chian School of Medicine</p>
</li>
<li>
<p><strong>Co-Lead PI: Jojn Chambers</strong>
<br>Lee Kong Chian School of Medicine</p>
</li>
<li>
<p><strong>Co-Lead PI: Sim Xueling</strong>
<br>National University of Singapore</p>
</li>
</ul>
</td>
<td rowspan="1" colspan="1">
<ol data-tight="true" class="tight">
<li>
<p>Conduct phenotypic associations of macronutrients with visceral adiposity
as primary outcome, and the visceral fat linked cardiometabolic traits
and diseases as secondary outcomes in multiethnic Asian population.</p>
</li>
<li>
<p>Perform GWAS of macronutrients in multiethnic Asian population using the
SG100K dataset and a GWAS meta-analysis using the UK Biobank macronutrient
intake data.</p>
</li>
<li>
<p>Perform functional annotation of significant loci and estimate the genetic
correlations of macronutrient intake with visceral fat linked cardiometabolic
traits and diseases as secondary outcomes.</p>
</li>
<li>
<p>Conduct one-sample and two-sample Mendelian Randomisation (MR) with macronutrient
intake as exposure variables and visceral adiposity as outcome variables,
with relevant sensitivity analyses.</p>
</li>
</ol>
</td>
</tr>
</tbody>
</table>
<p><strong><em>Institutions involved:</em></strong><em> Lee Kong Chian School of Medicine, National University Hospital</em>
</p>
<hr>
<h3><strong>A Structural Variation Catalogue Across Three Ancestrally Diverse Singapore Populations</strong></h3>
<table style="minWidth: 50px">
<colgroup>
<col>
<col>
</colgroup>
<tbody>
<tr>
<th rowspan="1" colspan="1">
<p>Team</p>
</th>
<th rowspan="1" colspan="1">
<p>Aims of Project</p>
</th>
</tr>
<tr>
<td rowspan="1" colspan="1">
<ul data-tight="true" class="tight">
<li>
<p><strong>Lead PI: Joanna Tan Hui Juan</strong>
<br>A*STAR Genome Institute of Singapore</p>
</li>
<li>
<p><strong>Co-Lead PI: Shyam Prabhakar</strong>
<br>A*STAR Genome Institute of Singapore</p>
</li>
<li>
<p><strong>Co-Lead PI: Patrick Tan Boon Ooi</strong>
<br>A*STAR Genome Institute of Singapore</p>
</li>
</ul>
</td>
<td rowspan="1" colspan="1">
<ol data-tight="true" class="tight">
<li>
<p>Build a catalogue of SVs (deletions, insertions, duplications, inversions,
translocations, and tandem repeats) from the PRECISE-SG100K dataset.</p>
</li>
<li>
<p>Investigate the identified SVs to uncover population-specific trends.</p>
</li>
<li>
<p>Examine the functional consequences of SVs in different genomic regions
as well as predict the impact of SVs in medically relevant genes.</p>
</li>
<li>
<p>Identify SVs that are associated with phenotypic traits within the PRECISE-SG100K
dataset.</p>
</li>
<li>
<p>Elucidate the impact of SVs on variation in cell type-specific gene expression
(SV-eQTLs) and validate SVs through copy number variation inferences from
scRNA-seq data.</p>
</li>
</ol>
</td>
</tr>
</tbody>
</table>
<p><strong><em>Institutions involved:</em></strong><em> A*STAR Genome Institute of Singapore</em>
</p>
<hr>
<h3><strong>Genome - Wide Association Study and Population - Based Evaluation of Patients with Diabetic Foot Ulcers</strong></h3>
<table style="minWidth: 50px">
<colgroup>
<col>
<col>
</colgroup>
<tbody>
<tr>
<th rowspan="1" colspan="1">
<p>Team</p>
</th>
<th rowspan="1" colspan="1">
<p>Aims of Project</p>
</th>
</tr>
<tr>
<td rowspan="1" colspan="1">
<ul data-tight="true" class="tight">
<li>
<p><strong>Lead PI: Joseph Lo</strong>
<br>Woodlands Health</p>
</li>
<li>
<p><strong>Co-Lead PI: Kavita Venkataraman</strong>
<br>National University of Singapore</p>
</li>
<li>
<p><strong>Co-Lead PI: Yusuf Ali</strong>
<br>Lee Kong Chian School of Medicine</p>
</li>
</ul>
</td>
<td rowspan="1" colspan="1">
<p>Primary aim:
<br>Identify genetic loci associated with diabetic foot ulcers in Asian patients
with diabetes mellitus.</p>
<p></p>
<p>Secondary aims:</p>
<ul data-tight="true" class="tight">
<li>
<p>Identify differences in genetic loci within Malay/Indian ethnicities.</p>
</li>
<li>
<p>Identify genetic loci associated with diabetic peripheral neuropathy.</p>
</li>
<li>
<p>Identify potential gene-environment interactions (for example, tobacco
smoking) associated with the risk ofdiabetic foot ulcers.</p>
</li>
<li>
<p>Identify socio-economical and other risk factors associated with diabetic
foot ulcers.</p>
</li>
<li>
<p>Identify correlations between macro-angiopathy, micro-vascular reactivity
nephropathy and retinopathy and diabetic foot ulcers.</p>
</li>
<li>
<p>Develop multi-polygenic risk score for developing diabetic foot ulcers.</p>
</li>
</ul>
</td>
</tr>
</tbody>
</table>
<p><strong><em>Institutions involved:</em></strong><em> Woodlands Health, National University of Singapore, Lee Kong Chian School of Medicine</em>
</p>
<hr>
<h3><strong>The SG100K_Cancer and Aging Workgroup: Developing Risk Models for Cancer Associations</strong></h3>
<table style="minWidth: 50px">
<colgroup>
<col>
<col>
</colgroup>
<tbody>
<tr>
<th rowspan="1" colspan="1">
<p>Team</p>
</th>
<th rowspan="1" colspan="1">
<p>Aims of Project</p>
</th>
</tr>
<tr>
<td rowspan="1" colspan="1">
<ul data-tight="true" class="tight">
<li>
<p><strong>Lead PI: Joanne Ngeow</strong>
<br>Lee Kong Chian School of Medicine</p>
</li>
<li>
<p><strong>Co-Lead PI: Rajkumar s/o Dorajoo</strong>
<br>A*STAR Genome Institute of Singapore</p>
</li>
<li>
<p><strong>Co-Lead PI: Neerja Karnani</strong>
<br>A*STAR Genome Institute of Singapore</p>
</li>
</ul>
</td>
<td rowspan="1" colspan="1">
<ol data-tight="true" class="tight">
<li>
<p>Generate common variant polygenic risk scores for common cancers (breast,
colorectal, liver, lung, and prostate cancers) and identify potential functional
rare coding genetic mutations in strong cancer related genes in the SG100K
dataset.</p>
</li>
<li>
<p>Generate additional age-related biomarkers (i.e as telomere length estimates)
related to cancer risk from the SG100K WGS data and identify genetic predispositions
associated with these biomarkers.</p>
</li>
<li>
<p>Linkage of genetic datasets with TRUST to derive clinical data and determine
common cancer status (breast, colorectal, liver, lung, and prostate cancers).</p>
</li>
</ol>
</td>
</tr>
</tbody>
</table>
<p><strong><em>Institutions involved:</em></strong><em> Lee Kong Chian School of Medicine, A*STAR Genome Institute of Singapore, Bioinformatics Institute</em>
</p>
<hr>
<h3><strong>Genetic Susceptibility of Age - Related Hearing Loss</strong></h3>
<table style="minWidth: 50px">
<colgroup>
<col>
<col>
</colgroup>
<tbody>
<tr>
<th rowspan="1" colspan="1">
<p>Team</p>
</th>
<th rowspan="1" colspan="1">
<p>Aims of Project</p>
</th>
</tr>
<tr>
<td rowspan="1" colspan="1">
<ul data-tight="true" class="tight">
<li>
<p><strong>Lead PI: Liu Jianjun</strong>
<br>A*STAR Genome Institute of Singapore</p>
</li>
<li>
<p><strong>Co-Lead PI: Nicolas Bertin</strong>
<br>A*STAR Genome Institute of Singapore</p>
</li>
<li>
<p><strong>Co-Lead PI: Lim Weng Khong</strong>
<br>Duke-NUS Medical School</p>
</li>
</ul>
</td>
<td rowspan="1" colspan="1">
<ol data-tight="true" class="tight">
<li>
<p>Generate a SG100K genome wide TR variation catalogue and characterisation
their respective prevalence in Asian populations.</p>
</li>
<li>
<p>Characterise the contributions of TR variations to the aetiology of complex
neurological and neurocognitive disorders.</p>
</li>
</ol>
</td>
</tr>
</tbody>
</table>
<p><strong><em>Institutions involved:</em></strong><em> A*STAR Genome Institute of Singapore, Duke-NUS Medical School</em>
</p>
<hr>
<h3><strong>Evaluating the Promise and Perils of Glucagon-Like Peptide-1 (GLP-1) Receptor Agonist: A Deep Dive into Therapeutic Potentials and Adverse Effects</strong></h3>
<table style="minWidth: 50px">
<colgroup>
<col>
<col>
</colgroup>
<tbody>
<tr>
<th rowspan="1" colspan="1">
<p>Team</p>
</th>
<th rowspan="1" colspan="1">
<p>Aims of Project</p>
</th>
</tr>
<tr>
<td rowspan="1" colspan="1">
<ul data-tight="true" class="tight">
<li>
<p><strong>Lead PI: Huang Jian</strong>
<br>A*STAR Singapore Institute for Clinical Sciences and Bioinformatics Institute</p>
</li>
<li>
<p><strong>Co-Lead PI: Dennis Wan</strong>
<br>A*STAR Singapore Institute for Clinical Sciences and Bioinformatics Institute</p>
</li>
</ul>
</td>
<td rowspan="1" colspan="1">
<ol data-tight="true" class="tight">
<li>
<p>Investigate the effects of GLP-1 receptor agonist on various domains of
health outcomes using an observational study design</p>
</li>
<li>
<p>Identify the non-synonymous single nucleotide polymorphisms (SNPs) of
GLP-1 receptor agonist prescription and predict nsSNPs responsible for
the differential response to GLP-1 receptor agonist.</p>
</li>
<li>
<p>Provide genetic evidence for the therapeutic potentials and adverse effects
of GLP-1 receptor agonists by adopting a drug target Mendelian randomisation
design.</p>
</li>
</ol>
</td>
</tr>
</tbody>
</table>
<p><strong><em>Institutions involved:</em></strong><em> A*STAR Singapore Institute for Clinical Sciences</em>
</p>
<hr>
<h3><strong>Unravelling the Pathogenesis of Inflammatory Bowel Disease and Associated Immune - Mediated Disorders in the Singaporean Population</strong></h3>
<table style="minWidth: 50px">
<colgroup>
<col>
<col>
</colgroup>
<tbody>
<tr>
<th rowspan="1" colspan="1">
<p>Team</p>
</th>
<th rowspan="1" colspan="1">
<p>Aims of Project</p>
</th>
</tr>
<tr>
<td rowspan="1" colspan="1">
<ul data-tight="true" class="tight">
<li>
<p><strong>Lead PI: Sun</strong>ny Wong
<br>Lee Kong Chian School of Medicine</p>
</li>
<li>
<p><strong>Co-Lead PI: Anselm Mak</strong>
<br>National University of Singapore</p>
</li>
<li>
<p><strong>Co-Lead PI: Bernett Lee</strong>
<br>Lee Kong Chian School of Medicine</p>
</li>
</ul>
</td>
<td rowspan="1" colspan="1">
<ol data-tight="true" class="tight">
<li>
<p>Identify Genetic Variants Associated with IBD and Related Immune-Mediated
Disorders in the Multi-Ethnic Singaporean Population. Explore and characterise
both common and rare genetic variants linked to IBD, and spondyloarthropathies,
uveitis, Behcet's disease, psoriasis, and other related immune-mediated
conditions within the diverse Singaporean population. This comprehensive
genetic investigation aims to unravel the unique genetic landscape of this
disease cluster, considering the multi- ethnic composition of the population.</p>
</li>
<li>
<p>Investigate the Effects of Lifestyle Factors on IBD and Associated Diseases
Risk and Progression. Systematically examine the impact of lifestyle factors,
including diet, physical activity, sleep, stress, and mental health, on
the risk and progression of IBD and associated immune-mediated diseases.
This multifaceted investigation seeks to discern the intricate relationships
between lifestyle choices and disease outcomes, contributing valuable insights
for developing targeted interventions and improving patient wellbeing.</p>
</li>
<li>
<p>Delineate shared and distinct mechanisms underlying IBD, spondyloarthropathy,
uveitis, psoriasis, Behcet's and other related conditions. Elucidate the
shared and unique genetic and biological pathways driving IBD, spondyloarthropathy,
uveitis, psoriasis, Behcet's disease, and other related conditions. This
will provide critical insights into disease mechanisms to guide targeted
prevention and treatment strategies for this nexus of related diseases.</p>
</li>
<li>
<p>lucidate Potential Interactions Between Genetic and Environmental Influences
on This Disease Cluster. Uncover and elucidate potential interactions between
genetic factors and environmental influences in the development and progression
of IBD and related conditions. This integrated approach aims to provide
a nuanced understanding of how genetic predispositions and environmental
exposures collaboratively contribute to the manifestation of this disease
cluster, offering a foundation for personalised and precision medicine
strategies.</p>
</li>
</ol>
</td>
</tr>
</tbody>
</table>
<p><strong><em>Institutions involved:</em></strong><em> Lee Kong Chian School of Medicine, National University of Singapore</em>
</p>
<hr>
<h3><strong>Genetic of Allergic Diseases and Acne Vulgaris in the Singapore Population: Validation and Functional Characterisation of Candidates</strong></h3>
<table style="minWidth: 50px">
<colgroup>
<col>
<col>
</colgroup>
<tbody>
<tr>
<th rowspan="1" colspan="1">
<p>Team</p>
</th>
<th rowspan="1" colspan="1">
<p>Aims of Project</p>
</th>
</tr>
<tr>
<td rowspan="1" colspan="1">
<ul data-tight="true" class="tight">
<li>
<p><strong>Lead PI: Chew Fook Tim</strong>
<br>National University of Singapore</p>
</li>
</ul>
</td>
<td rowspan="1" colspan="1">
<ol data-tight="true" class="tight">
<li>
<p>Validate disease-associated genetic polymorphisms and enviromental factors
that were previously identified and functionally characterised in the SMCGES
cohort, using the PRECISE-SG100K dataset.</p>
</li>
<li>
<p>Investigate the associations of previously identified asthma/AR/AD/acne
candidate genes with the other clinical parameters relevant to the disease
of interest. For instance, whether the allelic/genotypic differences of
genetic variants would affect the treatment response, lung functon (spirometry),
skin condition (sites of flexural dermatitis and psoriasis, etc.) in complex
disease.</p>
</li>
<li>
<p>Reproduce and validate the observed associations between specific dietary
habits and allergic diseases, using a more extensive and culturally relevant
FFQ.</p>
</li>
<li>
<p>Explore causal relationship between dietary habits and allergic diseases
by understanding how changes in dietary patterns influence the development
and progression of allergic diseases.</p>
</li>
</ol>
</td>
</tr>
</tbody>
</table>
<p><strong><em>Institutions involved:</em></strong><em> National University of Singapore</em>
</p>
<hr>
<h3><strong>Modulation of Cholesterol 7α-hydroxylase (CYP7A1) Activity as an Orthogonal Approach to the Management of Hypercholesterolemia</strong></h3>
<table style="minWidth: 50px">
<colgroup>
<col>
<col>
</colgroup>
<tbody>
<tr>
<th rowspan="1" colspan="1">
<p>Team</p>
</th>
<th rowspan="1" colspan="1">
<p>Aims of Project</p>
</th>
</tr>
<tr>
<td rowspan="1" colspan="1">
<ul data-tight="true" class="tight">
<li>
<p><strong>Lead PI: Ho Han Kiat</strong>
<br>National University of Singapore</p>
</li>
</ul>
</td>
<td rowspan="1" colspan="1">
<ol data-tight="true" class="tight">
<li>
<p>Determine the prevalence of CYP7A1 single nucleotide polymorphisms (SNP)
locally, on extrapolation, to the region that presents similar ethnicities.</p>
</li>
<li>
<p>Ascertain the relationship between CYP7A1 SNPs and hypercholesterolemia
in our local population.</p>
</li>
<li>
<p>Identify the target population most likely to benefit from targeting CYP7A1
as an orthogonal approach to cholesterol control.</p>
</li>
<li>
<p>Study the impact of non-genetic extrinsic factors, such as comorbidities
and comedications, on the genotypes to discern the possibility of phenoconversion.</p>
</li>
</ol>
</td>
</tr>
</tbody>
</table>
<p><strong><em>Institutions involved:</em></strong><em> National University of Singapore</em>
</p>
<hr>
<h3><strong>Multi - Omics Data Analysis for Novel Depression Mechanisms using Deep Learning Tools</strong></h3>
<table style="minWidth: 50px">
<colgroup>
<col>
<col>
</colgroup>
<tbody>
<tr>
<th rowspan="1" colspan="1">
<p>Team</p>
</th>
<th rowspan="1" colspan="1">
<p>Aims of Project</p>
</th>
</tr>
<tr>
<td rowspan="1" colspan="1">
<ul data-tight="true" class="tight">
<li>
<p><strong>Lead PI: Mu Yuguang</strong>
<br>Nanyang Technological University</p>
</li>
<li>
<p><strong>Co-Lead PI: Bernett Lee</strong>
<br>Lee Kong Chian School of Medicine</p>
</li>
<li>
<p><strong>Co-Lead PI: Geoffrey Tan Chern-Yee</strong>
<br>Institute of Mental Health</p>
</li>
</ul>
</td>
<td rowspan="1" colspan="1">
<p>1.1 Build machine learning and deep learning models specific to Singaporean
demographics which aid in prediction of depression.</p>
<p>1.2 To identify gene by marker and gene by environment interactions predicting
depressive and anxiety symptomswith dietary, nutrient, metabolic, lifestyle,
sociodemographic and cognitive factors</p>
<p>1.3 Apply feature selection approaches to discover important environmental
and genetic features used by the machine learning and deep learning models
to predict depression to ensure the model is explainable and reasonable
to physicians.</p>
<p>1.4 Explore the potential for the depression machine learning models to
be deployed in clinical settings.</p>
<p>2.1 Construct a language model which could extract complex interrelations
between patient features under Singapore context.</p>
<p>2.2 Assess the capability of the language model in imputing missing features
of a patient when other features were provided to extrapolate patient features
in clinical settings.</p>
<p>2.3 Assess suitability of leveraging the trained language model for transfer
learning.</p>
</td>
</tr>
</tbody>
</table>
<p><strong><em>Institutions involved:</em></strong><em> Nanyang Technological University, Lee Kong Chian School of Medicine, Institute of Mental Health</em>
</p>
<hr>
<h3><strong>Asian - Specific Parkinson’s Disease - Linked Genetic Risk Variants and Systemic Clinical Outcomes</strong></h3>
<table style="minWidth: 50px">
<colgroup>
<col>
<col>
</colgroup>
<tbody>
<tr>
<th rowspan="1" colspan="1">
<p>Team</p>
</th>
<th rowspan="1" colspan="1">
<p>Aims of Project</p>
</th>
</tr>
<tr>
<td rowspan="1" colspan="1">
<ul data-tight="true" class="tight">
<li>
<p><strong>Lead PI: Tan Eng King</strong>
<br>National Neuroscience Institute</p>
</li>
<li>
<p><strong>Co-Lead PI: Thomas Welton</strong>
<br>Duke-NUS Medical School</p>
</li>
<li>
<p><strong>Co-Lead PI: Chan Ling Ling</strong>
<br>Duke-NUS Medical School</p>
</li>
</ul>
</td>
<td rowspan="1" colspan="1">
<p>1a. Determine the prevalence of Asian specific LRRK2 coding variants and
other PD risk genes (e.g., APOE, SNCA, GBA1, HLA alleles, etc) in Malays,
Indians and Chinese in the SG100k cohort.</p>
<p>1b. Investigate the association of PD risk genes with comorbidities (Diabetes,
Hypertension, Heart disease, Autoimmune diseases, Infectious diseases,
vaccination history etc) in carriers and compare the associations with
non-carriers.</p>
<ol start="2" data-tight="true" class="tight">
<li>
<p>Investigate the association with motor and non-motor features between
PD risk gene carriers and non-carriers based on the quantitative outcome
measures (e.g. eye, cognition, bone mass, etc).</p>
</li>
<li>
<p>Investigate the differences in MRI regional volumes, lesion burden and
tissue microstructure between carriers and non- carriers.</p>
</li>
</ol>
</td>
</tr>
</tbody>
</table>
<p><strong><em>Institutions involved:</em></strong><em> National Neuroscience Institute, Duke-NUS Medical School</em>
</p>
<hr>
<h3><strong>Physiological, Environmental and Genetic Determinants of Heterogeneity in Singaporeans’ Health Span</strong></h3>
<table style="minWidth: 50px">
<colgroup>
<col>
<col>
</colgroup>
<tbody>
<tr>
<th rowspan="1" colspan="1">
<p>Team</p>
</th>
<th rowspan="1" colspan="1">
<p>Aims of Project</p>
</th>
</tr>
<tr>
<td rowspan="1" colspan="1">
<ul data-tight="true" class="tight">
<li>
<p><strong>Lead PI: Neerja Karnani</strong>
<br>A*STAR Bioinformatics Institute</p>
</li>
<li>
<p><strong>Co-Lead PI: Joanne Ngeow</strong>
<br>Lee Kong Chian School of Medicine</p>
</li>
<li>
<p><strong>Co-Lead PI: Brian Kennedy</strong>
<br>National University of Singapore</p>
</li>
<li>
<p><strong>Co-Lead PI: Rajkumar s/o Dorajoo</strong>
<br>A*STAR Genome Institute of Singapore</p>
</li>
</ul>
</td>
<td rowspan="1" colspan="1">
<ol data-tight="true" class="tight">
<li>
<p>Investigate the stressors associated with aging and identify the factors
contributing to resilience.</p>
</li>
<li>
<p>Investigate gender-specific variations in aging stressors and assess the
influence of reproductive aging.</p>
</li>
<li>
<p>Examine the effects of Asian ethnicity on the aging process and healthspan.</p>
</li>
<li>
<p>Evaluate the pharmacogenomic effects of medications on lifespan and overall
health during aging</p>
</li>
</ol>
</td>
</tr>
</tbody>
</table>
<p><strong><em>Institutions involved:</em></strong><em> A*STAR Bioinformatics Institute and Genome Institute of Singapore, Lee Kong Chian School of Medicine, National University of Singapore</em>
</p>
<hr>
<h3><strong>Portability of Catalogued Polygenic Risk Scores Across Ancestrally Diverse Singaporean Populations</strong></h3>
<table style="minWidth: 50px">
<colgroup>
<col>
<col>
</colgroup>
<tbody>
<tr>
<th rowspan="1" colspan="1">
<p>Team</p>
</th>
<th rowspan="1" colspan="1">
<p>Aims of Project</p>
</th>
</tr>
<tr>
<td rowspan="1" colspan="1">
<ul data-tight="true" class="tight">
<li>
<p><strong>Lead PI: Pierre-Alexis Goy</strong>
<br>A*STAR Genome Institute of Singapore</p>
</li>
<li>
<p><strong>Co-Lead PI: Li Jingmei</strong>
<br>A*STAR Genome Institute of Singapore</p>
</li>
</ul>
</td>
<td rowspan="1" colspan="1">
<ol data-tight="true" class="tight">
<li>
<p>Integration of Research Phenotypes with PRS Catalogue Ontologies and Identification
of Applicable PRS Models:</p>
<ul data-tight="true" class="tight">
<li>
<p>Align research phenotypes with the trait ontologies in the EBI-maintained
PRS catalogue.</p>
</li>
<li>
<p>Systematically select pertinent PRS models based on the relevance to mapped
research phenotypes.</p>
</li>
</ul>
</li>
<li>
<p>Assessment of Performance Across Diverse Ancestries within SG100K:</p>
<ul data-tight="true" class="tight">
<li>
<p>Evaluate the effectiveness of selected PRS models across the major ancestries
represented in the SG100K dataset (i.e. distribution, discrimination, calibration).</p>
</li>
<li>
<p>Analyse and compare performance metrics to identify any ancestry-specific
nuances in predictive accuracy.</p>
</li>
<li>
<p>Recommendations and portability assessment of EBI-catalogued published
PRS in a Singapore context.</p>
</li>
</ul>
</li>
</ol>
</td>
</tr>
</tbody>
</table>
<p><strong><em>Institutions involved:</em></strong><em> A*STAR Genome Institute of Singapore</em>
</p>
<hr>
<h3><strong>Advancing Asian - Centric Liver Disease Treatment: Machine Learning Applications in MASLD and MetALD Precision Medicine</strong></h3>
<table style="minWidth: 50px">
<colgroup>
<col>
<col>
</colgroup>
<tbody>
<tr>
<th rowspan="1" colspan="1">
<p>Team</p>
</th>
<th rowspan="1" colspan="1">
<p>Aims of Project</p>
</th>
</tr>
<tr>
<td rowspan="1" colspan="1">
<ul data-tight="true" class="tight">
<li>
<p><strong>Lead PI: Tan Nguan Soon</strong>
<br>Lee Kong Chian School of Medicine</p>
</li>
<li>
<p><strong>Co-Lead PI: Yew Kuo Chao</strong>
<br>Tan Tock Seng Hospital</p>
</li>
<li>
<p><strong>Co-Lead PI: Cheng Hong Sheng</strong>
<br>Lee Kong Chian School of Medicine</p>
</li>
</ul>
</td>
<td rowspan="1" colspan="1">
<ol data-tight="true" class="tight">
<li>
<p>Investigate the genomic risk of MASLD and associated metabolic traits
in Asian populations.</p>
</li>
<li>
<p>Interrogate the contribution of dietary components, alcohol intake and
physical activity to MASLD and MetALD disease spectrum.</p>
</li>
<li>
<p>Develop machine learning frameworks for risk stratification and identification
of predictive markers for MASLD disease spectrum.</p>
</li>
</ol>
</td>
</tr>
</tbody>
</table>
<p><strong><em>Institutions involved:</em></strong><em> Lee Kong Chian School of Medicine, Tan Tock Seng Hospital</em>
</p>
<hr>
<h3><strong>Unravelling the Correlation between Sarcopenia with Lifestyle, Genetics, and Comorbid diseases</strong></h3>
<table style="minWidth: 50px">
<colgroup>
<col>
<col>
</colgroup>
<tbody>
<tr>
<th rowspan="1" colspan="1">
<p>Team</p>
</th>
<th rowspan="1" colspan="1">
<p>Aims of Project</p>
</th>
</tr>
<tr>
<td rowspan="1" colspan="1">
<ul data-tight="true" class="tight">
<li>
<p><strong>Lead PI: Teh Bin Tean</strong>
<br>National Cancer Centre</p>
<p>Singapore</p>
</li>
<li>
<p><strong>Co-Lead PI: Frederick Koh Hong Xiang</strong>
<br>SingHealth</p>
</li>
</ul>
</td>
<td rowspan="1" colspan="1">
<ol data-tight="true" class="tight">
<li>
<p>Validate the multi-omics signature of people with various stages of sarcopenia.</p>
</li>
<li>
<p>Identify the influence of sarcopenia on comorbidities and its impact on
clinically relevant outcomes.</p>
</li>
<li>
<p>Evaluate correlation of biomarkers of sarcopenia with social economic
status</p>
</li>
</ol>
</td>
</tr>
</tbody>
</table>
<p><strong><em>Institutions involved:</em></strong><em> National Cancer Centre Singapore, SingHealth</em>
</p>
<hr>
<h3><strong>Young - Onset Obesity and Determinants of Cancer Prevalence</strong></h3>
<table style="minWidth: 50px">
<colgroup>
<col>
<col>
</colgroup>
<tbody>
<tr>
<th rowspan="1" colspan="1">
<p>Team</p>
</th>
<th rowspan="1" colspan="1">
<p>Aims of Project</p>
</th>
</tr>
<tr>
<td rowspan="1" colspan="1">
<ul data-tight="true" class="tight">
<li>
<p><strong>Lead PI: Yusuf Ali</strong>
<br>Lee Kong Chian School of Medicine</p>
</li>
<li>
<p><strong>Co-Lead PI: Sunny Wong</strong>
<br>Lee Kong Chian School of Medicine</p>
</li>
<li>
<p><strong>Co-Lead PI: Fan Xiuyi</strong>
<br>Lee Kong Chian School of Medicine</p>
</li>
</ul>
</td>
<td rowspan="1" colspan="1">
<ol data-tight="true" class="tight">
<li>
<p>Correlates of obesity before age 45 and incidence of cancer (EHR). Compare
risk relationships by ethnicity and sex.</p>
</li>
<li>
<p>Develop a dietary pattern score characterising inflammatory potential
of diet. Relate this score to circulating markers of inflammation and metabolic
health among PRECISE-SG100K participants under age 45.</p>
</li>
<li>
<p>Determine whether medications and bariatric surgery mitigate cancer risk
in young onset obese PRECISE-SG100K participants. Compare effectiveness
across ethnic groups and cancer sites.</p>
</li>
</ol>
</td>
</tr>
</tbody>
</table>
<p><strong><em>Institutions involved:</em></strong><em> Lee Kong Chian School of Medicine</em>
</p>
<hr>
<h3><strong>Implications of Alternative Splicing of Voltagegated Calcium Channels in Schizophrenia</strong></h3>
<table style="minWidth: 50px">
<colgroup>
<col>
<col>
</colgroup>
<tbody>
<tr>
<th rowspan="1" colspan="1">
<p>Team</p>
</th>
<th rowspan="1" colspan="1">
<p>Aims of Project</p>
</th>
</tr>
<tr>
<td rowspan="1" colspan="1">
<ul data-tight="true" class="tight">
<li>
<p><strong>Lead PI: Soong Tuck Wah</strong>
</p>
</li>
</ul>
</td>
<td rowspan="1" colspan="1">
<p>Profile the frequency of splicing associated genetic variations of VGCCs
and their auxiliary subunits, and their potential association with schizophrenia
in the SG100K cohort.</p>
</td>
</tr>
</tbody>
</table>
<p><strong><em>Institutions involved:</em></strong><em> National University of Singapore</em>
</p>
<hr>
<h3><strong>Exploring the Impact and Origins of Somatic Mutagenesis in Cardiovascular Disease</strong></h3>
<table style="minWidth: 50px">
<colgroup>
<col>
<col>
</colgroup>
<tbody>
<tr>
<th rowspan="1" colspan="1">
<p>Team</p>
</th>
<th rowspan="1" colspan="1">
<p>Aims of Project</p>
</th>
</tr>
<tr>
<td rowspan="1" colspan="1">
<ul data-tight="true" class="tight">
<li>
<p><strong>Lead PI: Tan Kar-Tong</strong>
</p>
</li>
</ul>
</td>
<td rowspan="1" colspan="1">
<ol data-tight="true" class="tight">
<li>
<p>Assess the impact of genetic variations the rate of somatic mutagenesis,
and the risk of CVDs.</p>
</li>
<li>
<p>Assess the impact of environmental exposures on the rate of somatic mutagenesis,
and the risk of CVDs.</p>
</li>
<li>
<p>Assess the impact of DNA damaging drugs on the rate of somatic mutagenesis
and CVDs.</p>
</li>
</ol>
</td>
</tr>
</tbody>
</table>
<p><strong><em>Institutions involved:</em></strong><em> National University of Singapore</em>
</p>
<hr>
<h3><strong>Alport Syndrome in the Singapore Population: An Under - Recognised Kidney Disease?</strong></h3>
<table style="minWidth: 50px">
<colgroup>
<col>
<col>
</colgroup>
<tbody>
<tr>
<th rowspan="1" colspan="1">
<p>Team</p>
</th>
<th rowspan="1" colspan="1">
<p>Aims of Project</p>
</th>
</tr>
<tr>
<td rowspan="1" colspan="1">
<ul data-tight="true" class="tight">
<li>
<p><strong>Lead PI: Ng Kar Hui</strong>
<br>National University of Singapore</p>
</li>
<li>
<p><strong>Co-Lead PI: David Bryce Matchar</strong>
<br>Duke-NUS Medical School</p>
</li>
<li>
<p><strong>Co-Lead PI: Jason Choo Chon Jun</strong>
<br>Duke-NUS Medical School</p>
</li>
</ul>
</td>
<td rowspan="1" colspan="1">
<p>1.Determine the prevalence of autosomal dominant (AD), X-linked (XL),
autosomal recessive (AR) and digenic Alport syndrome in Singapore; and
differences in these prevalences among the Chines, Malay and Indian populations
in Singapore.</p>
<p>2a. Determine the penetrance of Kidney, eye and hearing phenotypes in
AD Alport, XL male Alport and XL female Alport syndrome, stratified according
to age groups and gender in Singapore.</p>
<p>2b. Estimate the number of diagnosed Alport and versus mis-diagnosed or
undiagnosed Alport cases in Singapore and the differences in healthcare
costs, service utilisation and patterns of care among these groups.</p>
<p>2c. Correlate the severity of the kidney phenotypes with the genotypes
in COL4A3 and COL4A4, specifically comparing collagenous domain glycine
missense variants with other types of genetic variants.</p>
<p>3a. Determine the clinical features that predict an Alport genetic diagnosis.</p>
<p>3b. Determine the added risk of AD Alport on bad kidney outcomes (ESKD,
rapid GFR decline or heavy proteinuria).</p>
<p>3c. Determine the clinical features that predict a poor kidney outcome
in AD Alport subjects.</p>
</td>
</tr>
</tbody>
</table>
<p><strong><em>Institutions involved:</em></strong><em> National University of Singapore, Duke-NUS Medical School</em>
</p>
<hr>
<h3><strong>Risk Prediction for Congenital and Early - Onset Hearing Loss</strong></h3>
<table style="minWidth: 50px">
<colgroup>
<col>
<col>
</colgroup>
<tbody>
<tr>
<th rowspan="1" colspan="1">
<p>Team</p>
</th>
<th rowspan="1" colspan="1">
<p>Aims of Project</p>
</th>
</tr>
<tr>
<td rowspan="1" colspan="1">
<ul data-tight="true" class="tight">
<li>
<p><strong>Lead PI: Joshua Tay</strong>
<br>National University of Singapore</p>
</li>
<li>
<p><strong>Co-Lead PI: Tan Ene Choo</strong>
<br>KK Women's &amp; Children's Hospital</p>
</li>
<li>
<p><strong>Co-Lead PI: Goh Xueying</strong>
<br>National University of Singapore</p>
</li>
</ul>
</td>
<td rowspan="1" colspan="1">
<ol data-tight="true" class="tight">
<li>
<p>Describe the genetic landscape of congenital and early-onset hearing loss
in multi-ethnic Singapore. Identify the prevalence of known genetic variants
and novel variants associated with hearing loss.</p>
</li>
<li>
<p>Analyse interactions between hearing loss-associated genetic variants
and clinical events that may potentiate hearing loss (e.g. use of ototoxic
drugs).</p>
</li>
<li>
<p>Develop and validate a polygenic risk score for congenital and early onset
hearing loss based on an individual's genotype.</p>
</li>
</ol>
</td>
</tr>
</tbody>
</table>
<p><strong><em>Institutions involved:</em></strong><em> National University of Singapore, KK Women’s &amp; Children’s Hospital, National University Hospital</em>
</p>
<hr>
<h3><strong>Biological Age Clocks for Multiple Organ Systems and the Lifestyle and Genetic Risk Factors of Advanced Biological Age</strong></h3>
<table style="minWidth: 50px">
<colgroup>
<col>
<col>
</colgroup>
<tbody>
<tr>
<th rowspan="1" colspan="1">
<p>Team</p>
</th>
<th rowspan="1" colspan="1">
<p>Aims of Project</p>
</th>
</tr>
<tr>
<td rowspan="1" colspan="1">
<ul data-tight="true" class="tight">
<li>
<p><strong>Lead PI: Andrea B. Maier</strong>
<br>National University of Singapore</p>
</li>
<li>
<p><strong>Co-Lead PI: Weilan Wang</strong>
<br>National University of Singapore</p>
</li>
</ul>
</td>
<td rowspan="1" colspan="1">
<ol data-tight="true" class="tight">
<li>
<p>Investigate the optimal versus current reference ranges of organ systems
against the risk of age-related diseases using Singaporean data.</p>
</li>
<li>
<p>Develop and validate a biological clock on organ systems (cardiovascular,
pulmonary, metabolic, immune, hepatic, and musculoskeletal systems) based
on Singaporean data.</p>
</li>
<li>
<p>Explore the lifestyle and genetic risk factors associated with advanced
biological organ age.</p>
</li>
</ol>
</td>
</tr>
</tbody>
</table>
<p><strong><em>Institutions involved:</em></strong><em> National University of Singapore</em>
</p>
<hr>
<h3><strong>Identification of Risk Factors for Gastrointestinal Cancers through Analysis of Genetic and Phenotypic Data</strong></h3>
<table style="minWidth: 50px">
<colgroup>
<col>
<col>
</colgroup>
<tbody>
<tr>
<th rowspan="1" colspan="1">
<p>Team</p>
</th>
<th rowspan="1" colspan="1">
<p>Aims of Project</p>
</th>
</tr>
<tr>
<td rowspan="1" colspan="1">
<ul data-tight="true" class="tight">
<li>
<p><strong>Lead PI: Patrick Tan</strong>
<br>Duke-NUS Medical School</p>
</li>
<li>
<p><strong>Co-Lead PI: Lim Weng Khong</strong>
<br>Duke-NUS Medical School</p>
</li>
<li>
<p><strong>Co-Lead PI: Rajkumar s/o Dorajoo</strong>
<br>A*STAR Genome Institute of Singapore</p>
</li>
</ul>
</td>
<td rowspan="1" colspan="1">
<ol data-tight="true" class="tight">
<li>
<p>Systematically identify genes associated with gastrointestinal cancer
risks and survival and estimate penetrance (the cancer risk associated
with gene variants) of both novel and known pathogenic genes in gastrointestinal
cancer.</p>
</li>
<li>
<p>Investigate the interactions between the human genome, lifestyle factors
and presence of precursor lesions. This aim to determine the extent that
a healthier lifestyle can mitigate gastrointestinal cancer risk in subjects
with premalignant lesions or carrying a cancer predisposition gene.</p>
</li>
<li>
<p>Quantify the proportional contribution of human genome and lifestyle factors
to risks and survival outcomes of gastrointestinal cancers. We also aim
to develop predictive models for gastrointestinal risks by integrating
these factors.</p>
</li>
</ol>
</td>
</tr>
</tbody>
</table>
<p><strong><em>Institutions involved:</em></strong><em> Duke-NUS Medical School, A*STAR Genome Institute of Singapore</em>
</p>
<hr>
<h3><strong>Genomic Associations of COVID - 19 Susceptibility &amp; Severity in Singapore</strong></h3>
<table style="minWidth: 50px">
<colgroup>
<col>
<col>
</colgroup>
<tbody>
<tr>
<th rowspan="1" colspan="1">
<p>Team</p>
</th>
<th rowspan="1" colspan="1">
<p>Aims of Project</p>
</th>
</tr>
<tr>
<td rowspan="1" colspan="1">
<ul data-tight="true" class="tight">
<li>
<p><strong>Lead PI: Kelvin Bryan Tan</strong>
<br>Ministry of Health</p>
</li>
</ul>
</td>
<td rowspan="1" colspan="1">
<ol data-tight="true" class="tight">
<li>
<p>Mine and catalogue published disease severity and susceptibility genetic
variants and testing their prevalence in Singaporeans. This will include
not just previously published variants but identify potential new variants
which are associated with COVID severity, susceptibility and long COVID
outcomes. Prevalence and allele frequencies of these variants will be further
studied.</p>
</li>
<li>
<p>Genome wide association study (GWAS) to identify novel host genetic factors
in our population.</p>
</li>
<li>
<p>Assessing and developing genetic risk scores of disease severity, susceptibility
and long COVID outcomes in our population.</p>
</li>
</ol>
</td>
</tr>
</tbody>
</table>
<p><strong><em>Institutions involved:</em></strong><em> Ministry of Health</em>
</p>
<p></p>