---
title: "PRECISE: SG100K Driver Projects"
permalink: /precise-sg100k-driver-projects/
variant: tiptap
description: ""
third_nav_title: Our Data Users
---
<h4><strong>Computation of genome - wide LD scores and matrices from the SG100K resource</strong></h4>
<table style="minWidth: 50px">
<colgroup>
<col>
<col>
</colgroup>
<tbody>
<tr>
<th rowspan="1" colspan="1">
<p>Team</p>
</th>
<th rowspan="1" colspan="1">
<p>Aims of Project</p>
</th>
</tr>
<tr>
<td rowspan="1" colspan="1">
<ul data-tight="true" class="tight">
<li>
<p><strong>Lead PI: Li Jingmei</strong>
<br>A*STAR Genome Institute of Singapore</p>
</li>
<li>
<p><strong>Co-Lead PI: Rajkumar s/o Dorajoo</strong>
<br>A*STAR Genome Institute of Singapore</p>
</li>
<li>
<p><strong>Co-Lead PI: Khor Chiea Chuen </strong>A*STAR Genome Institute
of Singapore</p>
</li>
</ul>
</td>
<td rowspan="1" colspan="1">
<ol data-tight="true" class="tight">
<li>
<p>Taking reference from similar work performed by the Pan-UK Biobank, the
team will compute in-sample dosage-based LD matrices and scores for each
of the three major ancestry groups in SG100K:
<br>- LD score regression analysis to estimate heritabilities
<br>- Fine-mapping analysis to identify causal variants of well-powered complex
traits</p>
</li>
<li>
<p>The plan is to make the LD matrices available in Hail's BlockMatrix format
or similar. LD scores are also made available in LDSC-compatible flat files
(.l2.ldscore.gz and .M_5_50).</p>
</li>
</ol>
</td>
</tr>
</tbody>
</table>
<p><strong>Institutions involved:</strong> A*STAR Genome Institute of Singapore</p>
<hr>
<h4><strong>Chronic liver disease is a significant risk factor for adverse cardiometabolic outcomes</strong></h4>
<table style="minWidth: 50px">
<colgroup>
<col>
<col>
</colgroup>
<tbody>
<tr>
<th rowspan="1" colspan="1">
<p>Team</p>
</th>
<th rowspan="1" colspan="1">
<p>Aims of Project</p>
</th>
</tr>
<tr>
<td rowspan="1" colspan="1">
<ul data-tight="true" class="tight">
<li>
<p><strong>Lead PI: Mark Chan</strong>
<br>National University Hospital
<br>Cardiology</p>
</li>
<li>
<p><strong>Co-Lead PI: Dr Nicholas Chew</strong>
<br>National University Hospital
<br>Cardiology</p>
</li>
</ul>
</td>
<td rowspan="1" colspan="1">
<ol data-tight="true" class="tight">
<li>
<p>Investigate associations between established non-invasive Chronic Liver
Disease (CLD) biomarkers and cardiometabolic outcomes.</p>
</li>
<li>
<p>Evaluate how these associations relate to major adverse cardiac events.</p>
</li>
<li>
<p>Examine whether these associations with CLD are independent from associated
metabolic disease.</p>
</li>
</ol>
</td>
</tr>
</tbody>
</table>
<p><strong>Institutions involved:</strong> National University Hospital</p>
<hr>
<h4><strong>Nonlinear methods for genomic association analysis of eye diseases</strong></h4>
<table style="minWidth: 50px">
<colgroup>
<col>
<col>
</colgroup>
<tbody>
<tr>
<th rowspan="1" colspan="1">
<p>Team</p>
</th>
<th rowspan="1" colspan="1">
<p>Aims of Project</p>
</th>
</tr>
<tr>
<td rowspan="1" colspan="1">
<ul data-tight="true" class="tight">
<li>
<p><strong>Lead PI: Liu Dianbo</strong>
<br>National University of Singapore
<br>Ophthamology</p>
</li>
<li>
<p><strong>Co-Lead PI: A/P Wee Hwee Lin</strong>
<br>National University of Singapore</p>
</li>
<li>
<p><strong>Co-Lead PI: Dr Nicolas Bertin</strong>
<br>A*STAR Genome Institute of Singapore</p>
</li>
</ul>
</td>
<td rowspan="1" colspan="1">
<ol data-tight="true" class="tight">
<li>
<p>Identify non-linear genetic associations contributing to \ the susceptibility
and manifestation of diverse eye diseases.</p>
</li>
<li>
<p>Explore epistatic interactions and allelic heterogeneity within the genomic
data to unravel the complex relationships between multiple genetic variants.</p>
</li>
<li>
<p>Investigate how non-linear responses to environmental variables contribute
to the phenotypic variation, with a focus on refining our understanding
of gene- environment interactions in the context of ocular health.</p>
</li>
<li>
<p>Investigate and interpret the biological relevance of non-linear genetic
associations. Aim to gain insights into the underlying mechanisms linking
identified genetic variants to specific eye diseases and contribute to
a more comprehensive understanding of the biology involved.</p>
</li>
<li>
<p>Evaluate the public health implications of the identified non-linear genetic
associations, considering their potential impact on disease prevention,
intervention, and personalised treatment strategies. Assess the translational
potential of the research findings to inform clinical practice, public
health policies, and contribute to advancements in precision medicine for
ocular health.</p>
</li>
</ol>
</td>
</tr>
</tbody>
</table>
<p><strong>Institutions involved:</strong> National University of Singapore</p>
<hr>
<h4><strong>Advancing the understanding of biological mechanisms influencing chronic inflammatory skin diseases</strong></h4>
<table style="minWidth: 50px">
<colgroup>
<col>
<col>
</colgroup>
<tbody>
<tr>
<th rowspan="1" colspan="1">
<p>Team</p>
</th>
<th rowspan="1" colspan="1">
<p>Aims of Project</p>
</th>
</tr>
<tr>
<td rowspan="1" colspan="1">
<ul data-tight="true" class="tight">
<li>
<p><strong>Lead PI: Yew Yik Weng</strong>
<br>National Skin Centre</p>
</li>
<li>
<p><strong>Co-Lead PI: Steven Thng Tien Guan</strong>
<br>National Skin Centre</p>
</li>
<li>
<p><strong>Co-Lead PI: Marie Loh</strong>
<br>Lee Kong Chian School of Medicine</p>
</li>
</ul>
</td>
<td rowspan="1" colspan="1">
<ol data-tight="true" class="tight">
<li>
<p>Identify host genetic factors associated with chronic inflammatory skin
diseases, specifically AD, psoriasis and chronic urticaria using genome
wide association and rare variant analyses among SG100K study participants,
taking advantage of whole genome sequence data and linkage to disease information
from national electronic health records (NEHR).</p>
</li>
<li>
<p>Examine the relationship between genetic variants and polygenic risk scores
(PRS) associated with skin phenotypes and real-world health data for skin
diseases (including diagnosis, onset, severity and treatment outcomes)
to identify genetic predictors of disease trajectories, complications and
co-morbidities and treatment outcomes using the TRUST dataset.</p>
</li>
</ol>
</td>
</tr>
</tbody>
</table>
<p><strong>Institutions involved:</strong> National skin Centre, Lee Kong
Chian School of Medicine, A*STAR Skin Research Institute of Singapore</p>
<hr>
<h4><strong>Mood and diet in patients with irritable bowel syndrome (IBS) in Singapore</strong></h4>
<table style="minWidth: 50px">
<colgroup>
<col>
<col>
</colgroup>
<tbody>
<tr>
<th rowspan="1" colspan="1">
<p>Team</p>
</th>
<th rowspan="1" colspan="1">
<p>Aims of Project</p>
</th>
</tr>
<tr>
<td rowspan="1" colspan="1">
<ul data-tight="true" class="tight">
<li>
<p><strong>Lead PI: Kuang Ziyang Jonathan</strong>
<br>Gastroenterology &amp; Hepatology, Tan Tock Seng Hospital</p>
</li>
<li>
<p><strong>Co-Lead PI: Sunny Wong</strong>
<br>Lee Kong Chian School of Medicine</p>
</li>
<li>
<p><strong>Co-Lead PI: Chen Kok Pun</strong>
<br>Gastroenterology &amp; Hepatology
<br>Tan Tock Seng Hospital</p>
</li>
</ul>
</td>
<td rowspan="1" colspan="1">
<ol data-tight="true" class="tight">
<li>
<p>Evaluate the dietary pattern and characteristics of patients with IBS
in Singapore.</p>
</li>
<li>
<p>Identify patterns of mood disorders in patients with IBS in Singapore.</p>
</li>
<li>
<p>Identify genetic variants associated with IBS in the multi-ethnic Singaporean
Population. Explore and characterise common genetic polymorphisms IBS within
the diverse Singaporean population. This comprehensive genetic investigation
aims to unravel the unique genetic landscape of IBS, considering the multi-ethnic
composition of the population.</p>
</li>
<li>
<p>Investigate the effects of lifestyle factors on IBS Risk and progression.
Systematically examine the impact of lifestyle factors, including diet,
physical activity, sleep, stress, and mental health, on the risk and progression
of IBS. This multifaceted investigation seeks to discern the intricate
relationships between lifestyle choices and IBS, contributing valuable
insights for developing targeted interventions and improving patient outcomes.</p>
</li>
<li>
<p>Elucidate potential interactions between genetic and environmental influences
on IBS. Uncover and elucidate potential interactions between genetic factors
and environmental influences in the development and progression of IBS.
This integrated approach aims to provide a nuanced understanding of how
genetic predispositions and environmental exposures collaboratively contribute
to the manifestation of IBS, offering a foundation for personalised and
precision medicine strategies.</p>
</li>
</ol>
</td>
</tr>
</tbody>
</table>
<p><strong>Institutions involved:</strong> National University Hospital</p>
<p></p>